src/inc/switches.h

```diff
@@ -141,7 +141,7 @@
 #endif
 
 // GCCoverage has a dependency on msvcdisXXX.dll, which is not available for CoreSystem. Hence, it is disabled for CoreSystem builds.
```

Comment:
Could you please update the comment to reflect the fact that we are enabling it for Linux too?